app/login/login.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component} from "@angular/core";
+import { FormBuilder, Validators, FormGroup } from "@angular/forms";
+import { RouterModule, Router } from '@angular/router';
+
+import { LoginService } from "./login.service";
+
+@Component({
+  selector: "login",
+  templateUrl: "app/login/login.component.html",
+  styleUrls:  ["app/login/login.component.css"]
+})
+export class LoginComponent {
+	loginForm :FormGroup;
+	user: {};
+
+    constructor(
+    	public fb: FormBuilder,
+    	private router: Router,
+    	private loginService: LoginService
```

Comment:
private variables are visible within the class only and during the compiling these are hide while using other classes and thus performance improves